src/SerialPort/SerialPort.ts

```diff
@@ -89,6 +89,9 @@ export const createSerialPort = async (
         ipcRenderer.invoke(SERIALPORT_CHANNEL.SET, path, newOptions);
     };
 
+    const getOptions = (): Promise<SerialPortOpenOptions<AutoDetectTypes>> =>
```

Comment:
In https://github.com/NordicSemiconductor/pc-nrfconnect-launcher/blob/b4e8c2e4b0941cc7a838010b1bc9dabedff25974/src/ipc/serialport.ts#L55 it says  the return value is `SerialPortOpenOptions<AutoDetectTypes> | undefined`, here just `SerialPortOpenOptions<AutoDetectTypes>`.

I think the `| undefined` needs to be added here too.

Reply:
Thanks, good catch :)